doc: Refer test commands section of BLE Core Spec in DTM sample (#19775)
nrfconnect/sdk-nrf: samples/bluetooth/direct_test_mode/README.rst

   :depth: 2

This sample enables the Direct Test Mode functions described in `Bluetooth® Core Specification <Bluetooth Core Specification_>`_ (Vol. 6, Part F).

> The actual encoding of the test commands and events are described in Vol. 6, Part F, section 3.3 of this specification document.

Reviewer: Just a question: Is it encoding of test commands and encoding of events? If yes, then ... **is** described.

Reviewer 2: I looked directly to the spec. The section 3.3 name is "Commands and events" but it describes the encoding of commands only. The actual bit encoding of events is described in the next section: 3.4. So I'd mention both sections here. Since we want to be very clear to the customers, I'd be specific that 3.3 is for commands and 3.4 for events and there should be no doubt.

> The actual encoding of the test commands and encoding of events are described in section 3.3 and section 3.4, respectively, of Vol. 6, Part F of this specification document.

Reviewer: ```suggestion
The actual encoding of the test commands and events are described in sections 3.3 and 3.4, respectively, of Vol. 6, Part F of this specification document.
```

> Also, the actual encoding of the test commands and events are described in Vol. 6, Part F, section 3.3 of this specification document.
> The `Vendor-specific packet payload`_ section describes some vendor-specific commands, which comply with the core specification.

Reviewer: ```suggestion
The actual encoding of the test commands and events are described in Vol. 6, Part F, section 3.3 of this specification document.
The `Vendor-specific packet payload`_ section describes some vendor-specific commands that comply with the core specification.
```
